test/ssl_test_ctx.h

```diff
@@ -60,6 +60,11 @@ typedef enum {
     SSL_TEST_HANDSHAKE_RENEGOTIATE
 } ssl_handshake_mode_t;
 
+typedef enum {
```

Comment:
; git grep 'typedef enum {' shows 28 instances, almost all in the CT code.  Most of the time we give the enum a name (yes, admittedlu, most are in the apps where I did that).